src/web_server.cpp

```diff
@@ -1090,6 +1193,9 @@ web_server_setup() {
   server.on("/saveemoncms$", handleSaveEmoncms);
   server.on("/savemqtt$", handleSaveMqtt);
   server.on("/saveadmin$", handleSaveAdmin);
+  server.on("/savetesla$", handleSaveTesla);
+  server.on("/saveteslavi$", handleSaveTeslaVI);
```

Comment:
As an FYI I am currently adding some code that will make these redundant, all new config will be saved by a POST to `/config` but I will update when I merge. This change means you just have to add two lines in `app_config.cpp' and that will the new config option.

Reply:
excellent

Reply:
Take a look at #76, will probably merge this PR first so I can resolve the conflicts